responses/en/HassRespond.yaml

```diff
@@ -0,0 +1,9 @@
+language: en
+responses:
+  intents:
+    HassRespond:
+      default: "Hello from Home Assistant."
```

Comment:
```suggestion
      default: "How can I help?"
```

Reply:
Can we call this something else than default? Or do we always need a default?

Reply:
Maybe say something about needing the wake word, since we don't support conversations yet?

Reply:
Nah, that will be a good time to learn it, and a good reason for us to add conversation support.